### PR TITLE
Adjust spacing, make the scrollbar always visible and other tweaks to match the design

### PR DESCRIPTION
**Pull request by kryswisnaskas**
**Description of change**
Adjusts spacing for the navigation pages to better match the design. The scrollbar for the  main activity reports table is always visible now to avoid confusion. Other minor tweaks to match with the design.

**How to test**
- pull changes
- verify scrollbar is visible, things work as before, but match the design better

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/0

**Checklist**

- [ ] Meets issue criteria
- [ ] Code tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -137,7 +137,7 @@ function App() {
           <Header />
           <SiteNav admin={admin} authenticated={authenticated} logout={logout} user={user} />
           <div className="grid-row maxw-widescreen flex-align-start smart-hub-offset-nav tablet:smart-hub-offset-nav desktop:smart-hub-offset-nav margin-top-9">
-            <div className="grid-col-12 margin-top-2 margin-right-2">
+            <div className="grid-col-12 margin-top-2 margin-right-2 margin-left-3">
```
Comment (jasalisbury) [inline frontend/src/App.js:140]: I added the same class to just the landing page (`src/pages/Landing/index.js` around line 328) so the landing page is probably pushed a little to far to the right.

**Review by jasalisbury — APPROVED** [review]
:+1:

File: frontend/src/App.js (hunk 1 above)
Comment (kryswisnaskas) [reply frontend/src/App.js:140]: Sounds good.

**Review by rahearn — APPROVED** [review]
I don't see a vertical scrollbar on that table, but I'm on a mac with a trackpad and a huge screen, so I'm wondering if this is just macOS being real agressive about hiding scrollbars no matter what.

**Pull request merged**